### PR TITLE
[flex]: Add inspec tests

### PR DESCRIPTION
**Pull request by gavindidrichsen**
```rspec
hab pkg exec chef/inspec inspec exec /src/. --chef-license=accept -t docker://9953c9d425f0355049d8f7aa12046ace73a1793b47b226d1143048376f65f37d --input-file /src/attributes.yml

Profile: Habitat Core Plan flex (flex)
Version: 0.1.0
Target:  docker://9953c9d425f0355049d8f7aa12046ace73a1793b47b226d1143048376f65f37d

  ✔  core-plans-flex-exists: Ensure flex exists
     ✔  File /hab/pkgs/core/flex/2.6.4/20200602214726/bin/flex is expected to exist
  ✔  core-plans-flex-works: Ensure flex works as expected
     ✔  Command: `hab pkg path core/flex` exit_status is expected to eq 0
     ✔  Command: `hab pkg path core/flex` stdout is expected not to be empty
     ✔  Command: `DEBUG=true; hab pkg exec core/flex/2.6.4/20200602214726 flex --version` exit_status is expected to eq 0
     ✔  Command: `DEBUG=true; hab pkg exec core/flex/2.6.4/20200602214726 flex --version` stdout is expected not to be empty
     ✔  Command: `DEBUG=true; hab pkg exec core/flex/2.6.4/20200602214726 flex --version` stdout is expected to match /flex 2.6.4/
     ✔  Command: `DEBUG=true; hab pkg exec core/flex/2.6.4/20200602214726 flex --version` stderr is expected to be empty


Profile Summary: 2 successful controls, 0 control failures, 0 controls skipped
Test Summary: 7 successful, 0 failures, 0 skipped
+ set +x
[6][default:/src:0]# 
```

### CODE REVIEW

File: controls/flex_works.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  describe flex_path do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not be_empty }
+  end
+  
+  flex_pkg_ident = ((flex_path.stdout.strip).match /(?<=pkgs\/)(.*)/)[1]
+  describe command("DEBUG=true; hab pkg exec #{ flex_pkg_ident} flex --version") do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not be_empty }
+    its('stdout') { should match /flex 2.6.4/ }
```
Comment (skpaterson) [inline controls/flex_works.rb]: Same version issue here

Comment (gavindidrichsen) [reply controls/flex_works.rb]: fixed as agreed.

**Comment by gavindidrichsen** [comment]
Updated test to parse pkg_version from the pkg_ident according to comment [here](https://github.com/chef-base-plans/attr/pull/1#discussion_r435860936)

**Review by skpaterson — APPROVED** [review]
Looks good, thanks @gavindidrichsen

**Pull request merged**